tests/DependencyInjection/Compiler/RegisterMenusPassTest.php

```diff
@@ -21,93 +24,93 @@ class RegisterMenusPassTest extends TestCase
 
     protected function setUp(): void
     {
-        if (!\class_exists(ServiceClosureArgument::class)) {
-            $this->markTestSkipped('The RegisterMenuPass requires Symfony DI 3.3+.');
-        }
+        $this->containerBuilder = new ContainerBuilder();
+
+        $this->containerBuilder->register('knp_menu.menu_provider.lazy', LazyProvider::class)
+            ->setArgument(0, null);
+        $this->containerBuilder->register('knp_menu.menu_provider.builder_service', \stdClass::class);
```

Comment:
Shouldn't we assert at least once that `knp_menu.menu_provider.builder_service` definition is removed after the process method successfully executes?

Reply:
Removed by mistake on my previous edit. I will recover that assertion.

Reply:
Fixed.